README.md

```diff
@@ -1 +1,65 @@
-# SimpsonQuotes
\ No newline at end of file
+# RestorauntApi
+
+> The repo contains my javascript code for the Restoraunt Api project
+
+## Built With
+
+- Javascript
+- Html
+- Bootstrap
+- Css
+- Jest
+- Fontawesome CDN
+
+## LiveLink
+[LiveLink](https://igorkol91.github.io/RestorauntApi/dist/)
+
+### Video demo
+[VideoLink](https://drive.google.com/file/d/1oHNiJyfbs9XOWjcOHDLC5r5jhKV5Bno-/view?usp=sharing)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+In vsCode console run $git clone https://github.com/igorkol91/RestorauntApi
+Then run $cd (the folder name)
+Choose the corect branch by runnung $git checkout develop
+To install all dependancies in the console run $npm install
+To start the server run $npm start
```

Comment:
### Optional
- [ ] Please arrange this part so that it can be readable now its not easy to see every step

Reply:
Thanks, @DeliceLydia :pray:
This has been fixed and we will record a new video tomorrow.